app/controllers/api/v1/items_controller.rb

```diff
@@ -0,0 +1,16 @@
+class Api::V1::ItemsController < ApplicationController
+  def create
+    if item_params.keys.count == 4
+      item = Item.create(item_params)
```

Comment:
What should happen if the item can't be created (ie. it doesn't pass validations)?

Reply:
good point